packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp

```diff
@@ -1623,24 +1623,24 @@ namespace MueLu {
       TEUCHOS_TEST_FOR_EXCEPTION(partName != "zoltan" && partName != "zoltan2", Exceptions::InvalidArgument,
                                  "Invalid partitioner name: \"" << partName << "\". Valid options: \"zoltan\", \"zoltan2\"");
 
-#ifndef HAVE_MUELU_ZOLTAN
```

Comment:
Why the change of logic?

Reply:
whoops, good catch. I started typing out `#if !defined(HAVE_MUELU_ZOLTAN)`, and then backspaced it when I decided I didn't want to change all of them to `#if defined`, but replaced it incorrectly.